src/style/utils/query-theme.js

```diff
@@ -0,0 +1,28 @@
+
+const isUpper = char => char.toUpperCase() === char;
+const camelToDash = (acc, char) => `${acc}${isUpper(char) ? `-${char.toLowerCase()}` : char}`;
+const toCSSCase = str => str.split('').reduce(camelToDash, '');
```

Comment:
I think it's called kebab case :)

Reply:
it could be done more straightforward: `str.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();`